core/db/curibio/alembic/versions/d7a72568d90b_add_pulse3d_version_0_29.1_0.py

```diff
@@ -0,0 +1,26 @@
+"""add pulse3d version 0.29.1
```

Comment:
You can just change the upgrade script for 0.29.0 instead of adding this one. Also there is a typo in the name of this file